app/services/import_repository/parse_repository.rb

```diff
@@ -31,6 +31,26 @@ def has_too_many_rows?
       @sheet.last_row > Constants::IMPORT_REPOSITORY_ITEMS_LIMIT
     end
 
+    def check_file_for_duplicates
```

Comment:
Method name doesn't reflect what the method returns or does. Should probably be called just `duplicate_ids`